AdventOfCode2022/Program.cs

```diff
@@ -11,5 +66,120 @@ static void Main()
 
         for (int i = 0; i < textInput.Length; i++)
             textInput[i] = textInput[i].Trim();
+
+        List<int> coorH = new List<int> { 0, 0 };
+        List<int> coorT1 = new List<int> { 0, 0 };
+        List<int> coorT2 = new List<int> { 0, 0 };
+        List<int> coorT3 = new List<int> { 0, 0 };
+        List<int> coorT4 = new List<int> { 0, 0 };
+        List<int> coorT5 = new List<int> { 0, 0 };
+        List<int> coorT6 = new List<int> { 0, 0 };
+        List<int> coorT7 = new List<int> { 0, 0 };
+        List<int> coorT8 = new List<int> { 0, 0 };
+        List<int> coorT9 = new List<int> { 0, 0 };
```

Comment:
I would generally store such amount of variables in some kind of dictionary; list would be good there as well, coorH being zero index